02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -17,43 +17,41 @@ It is well-known among the PureScript community that its documentation is lackin
 Outline
 - Explain what "good" documentation is and define a criteria
 - Evaluate PureScript's documentation using that criteria
-- Explain why PureScript's documentation is lacking and what is being done to improve it
+- Explore where PureScript's documentation is lacking and what can be done to improve it
```

Comment:
When I hear "is being done" I hear "someone else is actively doing" which implies (to me) that my help isn't needed. I think an open-source project should communicate in a way which implies the project is a work-in-progress and it consists of a to-do list that anyone can work on.

I'll brainstorm some options as post a follow-up comment.

Reply:
- Explain why PureScript's documentation isn't developing as fast as wanted and describe how to improve it
- Explain why PureScript's documentation is lacking and explain the current development plans
- Explain why PureScript's documentation is lacking and develop plans to improve the current strategy

Reply:
> When I hear "is being done" I hear "someone else is actively doing" which implies (to me) that my help isn't needed.

Ah... good point. I read that, too, now that you bring that up.

Reply:
Thanks for the other ways to render this. It provides other points to consider.

> Explain why PureScript's documentation isn't _developing as fast as wanted_...

That is an issue in the documentation, but I think that is covered later in the document. I think 'lacking' is the best overall description because it doesn't explain what is lacking. Are we lacking documentation period? Is the current documentation lacking organization or clarity? It simply communicates things aren't the way they should be.

> describe how to improve it

'describe' is perhaps not the best word to use her. Perhaps 'suggest' because here are the conclusions we've made, but readers might have other ideas?

> explain the current development plans

This assumes there are such plans and without a mutually-held vision, I don't think it would be wise so say that.

> develop plans to improve the current strategy

Hmm... that's good rendering in at least one aspect. It sort of implies that we want people to get involved: "discuss and develop plans ..."

"the current strategy" implies there is one, and again I don't know if there is. Maybe it should just be "plans to improve the current situation."

Reply:
Good ideas. I've updated this part just now. I also broke it into two parts, because it seems to be the right thing to do here, but I'm not sure whether that'll fit the implementation of this outline later on in this document.